test/integration/ovs/ofctrl_test.go

```diff
@@ -584,7 +584,7 @@ func prepareFlows(table binding.Table) ([]binding.Flow, []*ExpectFlow) {
 			LoadRegToReg(0, 0, binding.Range{0, 15}, binding.Range{0, 15}).
 			LoadReg(0, 0x0ffe, binding.Range{16, 31}).
 			Done(). // Finish learn action.
-			Action().ResubmitToTable(table.GetID()).
+			Action().GotoTable(table.GetID()).
```

Comment:
Please confirm with the auther if he wants to resubmit/output the packet to the same table in this test case first. If yes, please keep "ResubmitToTable" as the final action.

Reply:
I'll use ResubmitToTable for learn flow. Thanks for your review.